net-misc/minidlna/minidlna-1.2.1.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A server software compliant with multiple clients"
+DESCRIPTION="MiniDLNA MiniDLNA (aka ReadyDLNA) is server software with the \
+aim of being fully compliant with DLNA/UPnP-AV clients."
+HOMEPAGE="http://sourceforge.net/projects/minidlna/"
```

Comment:
Oops, I forgot to suggest something for `HOMEPAGE`:
* _a)_
```diff
-HOMEPAGE="https://sourceforge.net/projects/minidlna/"
+HOMEPAGE="http://minidlna.sourceforge.net/
+	https://sourceforge.net/projects/minidlna/"
```

Other suggestions:

* _b)_
```
HOMEPAGE="https://sourceforge.net/projects/minidlna/"
```

* _c)_
```
HOMEPAGE="http://minidlna.sourceforge.net/"
```
(I don't know which is best. _b_ has a link to _c_, and _c_ has a link to _b_ :)